print/print.env

```diff
@@ -1,7 +1,7 @@
-INTERNAL_API_ROOT_URL=http://caddy:3001
+INTERNAL_API_ROOT_URL=http://caddy:3000/api
 FRONTEND_URL=http://localhost:3000
 SENTRY_PRINT_DSN=
 SENTRY_ENVIRONMENT=http://localhost:3000
 BROWSER_WS_ENDPOINT=ws://browserless:3000
 PRINT_URL=http://print:3003
-COOKIE_PREFIX=localhost_
\ No newline at end of file
+COOKIE_PREFIX=localhost_
```

Comment:
I think in #3262 you can also remove this variable. Or am I overlooking something?